src/main/java/org/jabref/gui/externalfiles/FileIgnoreUnlinkedFiles.java

```diff
@@ -0,0 +1,53 @@
+package org.jabref.gui.externalfiles;
+
+import java.io.BufferedReader;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.jabref.logic.l10n.Localization;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public enum FileIgnoreUnlinkedFiles {
+    DEFAULT(Localization.lang("Default")),
+    INCLUDE_ALL(Localization.lang("Include All Files"));
+
+    private static FileIgnoreUnlinkedFiles instance;
+    private static String gitIgnorePath = ".gitignore";
+    private static Set<String> IgnoreFileSet = new HashSet<>();
+    private static final Logger LOGGER = LoggerFactory.getLogger(FileIgnoreUnlinkedFiles.class);
+
+    private final String fileIgnoreOption;
+    FileIgnoreUnlinkedFiles(String fileIgnoreOption) {
+        this.fileIgnoreOption = fileIgnoreOption;
+    }
+
+    public String getFileIgnoreOption() {
+        return fileIgnoreOption;
+    }
+
+    public static Set<String> getIgnoreFileSet() {
```

Comment:
I'm really not a fan of these static methods. Keep the scope small and only use statics or globals if really neccessary. I think this method rather belongs in the UnlinkedFilesDialogViewModel